docs/navigation-prop.md

```diff
@@ -6,25 +6,23 @@ sidebar_label: Navigation prop
 
 Each `screen` component in your app is provided with the `navigation` prop automatically. It looks like this:
 
-* `this.props.navigation`
-  * `navigate` - go to another screen, figures out the action it needs to take to do it
+* Navigating
+  * `navigate` - navigates to another screen
   * `goBack` - close active screen and move back in the stack
+  * `replace` - replace the current route with a new one
+  * `dispatch` - manually send a navigation action to the navigator. This is a lower level method that you generally will not need to use as methods like `navigate` and `goBack` will handle it for you.
+* Navigating with stack-based navigators like `StackNavigator`
+  * `push` - pushes a new screen onto a stack navigator and navigates to it
+  * `pop` - pop the current screen off of the stack navigator and navigate to the previous screen
+  * `popToTop` - pop all screens except the bottom-most off of the stack navigator and navigate to that screen
```

Comment:
> pop all screens except the bottom-most off of the stack navigator and navigate to that screen

I think it is enough to say "pop all screens except the bottom-most off of the stack navigator" since that's what it does